PayPalCheckout/src/main/java/com/paypal/android/checkout/paymentbutton/PayPalCreditButton.kt

```diff
@@ -0,0 +1,101 @@
+package com.paypal.android.checkout.paymentbutton
+
+import android.content.Context
+import android.content.res.TypedArray
+import android.os.Build
+import android.util.AttributeSet
+import androidx.annotation.RequiresApi
+import androidx.core.content.res.use
+import com.paypal.android.checkout.paymentbutton.error.createFormattedIllegalArgumentException
+import com.paypal.pyplcheckout.R
+
+/**
+ * PayPalCreditButton provides a PayPal Credit button with the ability to modify the [color], [shape],
+ * and [size].
+ *
+ * Setting up PayPalCreditButton within an XML layout:
+ * ```
+ * <PayPalCreditButton
+ *      android:id="@+id/paymentButton"
+ *      android:layout_width="match_parent"
+ *      android:layout_height="wrap_content" />
+ * ```
+ *
+ * Optionally you can provide the following attributes: `paypal_credit_color`,
+ * `payment_button_shape`, and `payment_button_size`.
+ *
+ */
+@RequiresApi(Build.VERSION_CODES.M)
+class PayPalCreditButton @JvmOverloads constructor(
```

Comment:
Should we have some tests for these button classes? Not sure exactly what approach we want to take for testing UI elements, but we should probably have some coverage on the default cases and functionality.